tests/sentry/api/endpoints/test_organization_api_keys.py

```diff
@@ -0,0 +1,193 @@
+from __future__ import absolute_import
+
+from django.core.urlresolvers import reverse
+
+from sentry.models import ApiKey
+from sentry.testutils import APITestCase
+
+DEFAULT_SCOPES = [
+    'project:read',
+    'event:read',
+    'team:read',
+    'org:read',
+    'member:read',
+]
+
+
+class OrganizationApiKeys(APITestCase):
+    def test_org_admin_can_access(self):
+        self.login_as(user=self.user)
+
+        organization = self.create_organization(name='foo', owner=self.user)
+
+        path = reverse(
+            'sentry-api-0-organization-api-keys',
+            args=[organization.slug]
+        )
+
+        resp = self.client.get(path)
+
+        assert resp.status_code == 200
+
+    def test_member_no_access(self):
+        self.login_as(user=self.user)
+        organization = self.create_organization(name='foo', owner=self.user)
+
+        user = self.create_user('bar@example.com')
+        self.create_member(
+            organization=organization,
+            user=user,
+            role='member',
+        )
+
+        path = reverse(
+            'sentry-api-0-organization-api-keys',
+            args=[organization.slug]
+        )
+
+        self.login_as(user)
+
+        resp = self.client.get(path)
+
+        assert resp.status_code == 403
+
+    def test_admin_can_access(self):
+        self.login_as(user=self.user)
+        organization = self.create_organization(name='foo', owner=self.user)
+
+        admin_user = self.create_user('admin@example.com', is_superuser=True)
+        self.create_member(
+            organization=organization,
+            user=admin_user,
+            role='admin',
+        )
+
+        path = reverse(
+            'sentry-api-0-organization-api-keys',
+            args=[organization.slug]
+        )
+
+        self.login_as(admin_user)
+
+        resp = self.client.get(path)
+
+        assert resp.status_code == 200
+
+    def test_org_admin_can_create(self):
+        self.login_as(user=self.user)
+
+        organization = self.create_organization(name='foo', owner=self.user)
+
+        path = reverse(
+            'sentry-api-0-organization-api-keys',
+            args=[organization.slug]
+        )
+
+        resp = self.client.post(path)
+
+        assert resp.status_code == 200
+        assert resp.data.get('label') == 'Default'
+
+    def test_member_no_create(self):
+        self.login_as(user=self.user)
+        organization = self.create_organization(name='foo', owner=self.user)
+
+        user = self.create_user('bar@example.com')
+        self.create_member(
+            organization=organization,
+            user=user,
+            role='member',
+        )
+
+        path = reverse(
+            'sentry-api-0-organization-api-keys',
+            args=[organization.slug]
+        )
+
+        self.login_as(user)
+
+        resp = self.client.post(path)
+
+        assert resp.status_code == 403
+
+
+class OrganizationApiKeyDetails(APITestCase):
```

Comment:
nit, but i think we usually split out the index / details tests into two separate files (it just makes it slightly easier to find the tests)